.github/workflows/main.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Calculate test coverage
         run: cargo tarpaulin --out Html
       - name: Archive code coverage results
-        uses: actions/upload-artifact@v1
+        uses: actions/upload-artifact@v3

```

Comment:
is there a specific reason for changing the version?

Reply:
The pipeline fails now because version 1 is outdated. Switching to version 3 fixes the problem.